doc/news/changes/minor/20200901AlexanderGrayver

```diff
@@ -0,0 +1,5 @@
+Added: python wrapper for GridTools::replicate_triangulation,
+more general version of the GridTools::merge_triangulations is
+implemented
+<br>
+(Alexander Grayver, 2020/09/01)
```

Comment:
You technically also introduce an incompatibility in `merge_triangulation` So can you add an entry there.